Modules/CodeTrek/Entities/CodeTrekApplicant.php

```diff
@@ -22,6 +23,16 @@ public function center()
         return $this->belongsTo(OfficeLocation::class, 'center_id');
     }
 
+    public function getDaysInCodetrekAttribute(self $applicant)
+    {
+        $internDate = Carbon::parse($applicant->internship_start_date);
```

Comment:
This name sounds more meaningful
```suggestion
        $internshipStartDate = Carbon::parse($applicant->internship_start_date);
```